powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/database/model/PowerAuthSignatureMetadata.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * PowerAuth Server and related software components
+ * Copyright (C) 2023 Wultra s.r.o.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package io.getlime.security.powerauth.app.server.database.model;
+
+import lombok.AllArgsConstructor;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.NoArgsConstructor;
+
+
+/**
+ * Concrete implementation of the SignatureMetadata interface for PowerAuth.
+ * Contains metadata parameters specific to PowerAuth Signature.
+ *
+ * @author Jan Dusil
+ */
+@AllArgsConstructor
+@NoArgsConstructor
+public class PowerAuthSignatureMetadata implements SignatureMetadata<String, String> {
```

Comment:
Could you please double-check and clarify dirty-check behavior for this type? Especially when missing `equals`. We have faced some issues even for enums when upgrading Hibernate.

Reply:
Moreover, shouldn't be the class or even the interface _serializable_? Because `SignatureEntity` is.